WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -133,7 +133,7 @@
 
     private OnMagicLinkRequestInteraction mListener;
     private String mToken = "";
-    private boolean mInhibitMagicLogin;
+    private boolean mShouldReturnToPreviousActivity;
```

Comment:
For someone new coming into the code base, `mInhibitMagicLogin` will not help them understand why we need that boolean set at all. Magic Links isn't the reason we set that boolean, it's because we need to make sure the stack isn't cleared so that we can go back to the previous activity.

Right now, whether or not the user uses magic links, it clears the stack. So the reader of the code will see this if the name includes magic links:
```
if (can't be magic link login) {
  sign in
  finish activity
} else { // can be magic link or username login
  sign in
  clear stack
  finish activity
}
```
Because this is the only place we're using the boolean as a switch, I think it's important that we focus on making it as clear as possible here.